azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -231,6 +234,7 @@ public KubernetesContainerizedImpl(final Props azkProps,
             DEFAULT_MEMORY_LIMIT_MULTIPLIER);
     this.memoryLimit = this.getResourceLimitFromResourceRequest(this.memoryRequest, this.memoryRequest,
         memoryLimitMultiplier);
+    this.memoryLimitByConf = this.memoryLimit;
```

Comment:
can we change this variable name to defaultMemoryLimit?

Reply:
I'm eliminating all cpu and memory limit instances in the class. The limit should always be determined dynamically from resource request.